src/components/Location.vue

```diff
@@ -25,9 +25,10 @@
 			<span v-show="country">{{ $t('privacy', 'Your data is located in:') }} <strong>{{ country }}</strong>.</span>
 			<span v-show="!country">{{ $t('privacy', 'The admin has not selected the location of the server yet.') }}</span>
 			<NcActions v-if="$is_admin">
-				<NcActionButton icon="icon-rename" @click="editLocation">
-					{{ t('privacy', 'Change data location') }}
```

Comment:
Should this change be brought upstream into the vue lib directly?

Reply:
was already done some time agohttps://github.com/nextcloud/nextcloud-vue/blob/master/src/components/NcActions/NcActions.vue#L940-L942